tests/test_splitter.py

```diff
@@ -106,3 +149,103 @@ def test_switchback_balanced_splitter_abc():
         counts = sampled_treatment.treatment.value_counts()
         assert counts.max() == 2
         assert counts.min() == 1
+
+
+def test_stratified(df_strata):
+    splitter = StratifiedClusteredSplitter(
```

Comment:
There is a lot of code repetition in here, we could extract a function cllaed assert_balanced_strata(df) and just call this function in the tests

Reply:
done! changed the assertion pipeline to test clusters-strata unique combinations, as this is what the splitter does.